### PR TITLE
Accesibility

### PR DESCRIPTION
**Pull request by razvanso**
1. Page titles - no issues
2. Image text alternatives** (`alt text`)
    - renamed files
    - cleaned up alt text

3. Text headings
    - Add a hidden **h3** heading to enforce document hierarchy

4. Color contrast - no issues
5. Resize - no issues
6. Interaction
    - added border on `button:focus` state

7. Moving content - no issues
8. Multimedia - no issues
9. The basic structure of the page
    - changed text color of alt text on dark background (footer)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -213,7 +215,7 @@ <h2 class="heading">About</h2>
             <li>HTML</li>
             <li>CSS</li>
           </ul>
-          <img class="icon-bg" src="images/ic_languages_mob.svg" alt="#" />
+          <img class="icon-bg" src="images/ic_languages_mob.svg" alt="" />
         </li>
         <li class="red about heading3">
           Frameworks
```
Comment (mahbubul14) [inline index.html]: - [ ] In the mobile version of your page, the following section is not matched. Currently, your page has more space on the right side of this section. Please fix it.
**Optional:**
- [ ] Your design has no bullet for the items. In the Figma design, there are bullet points for every li. Please work on that.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
l1 | l2

File: styles.css [hunk 2]
```diff
@@ -1004,3 +1004,11 @@ section.intro-section .page-brk-dp {
...
   display: flex;
 }
+
+button:focus {
+  border: 3px solid #e24e5b;
+}
+
+footer img[alt] {
+  color: #fff;
+}
```
Comment (mahbubul14) [inline styles.css:1019]: - [ ] In the left bottom corner of your page(footer), the following icon doesn't match with the design. Please make the icon a little bit bigger to match the design.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
f1 | f2

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @d2ndjim ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Please add the "Getting Started" section on your readme file. Why don't you add content and say how to run it, to them so your README is more informative and professional?

- [ ] Please check the comments under this review :arrow_down_small: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] The commit message should be written in the imperative tense and the first letter should be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] In your `README` file, kindly add your name instead of the `author1`.
a1


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: styles.css [hunk 3]
```diff
@@ -1004,3 +1009,15 @@ section.intro-section .page-brk-dp {
...
```
Comment (thecodechaser) [inline styles.css:1019]: - [ ] As requested by the previous reviewer the icon at the bottom of your page isn't matching with the design and you are using a different icon, Please use the correct icon and make sure it matches the design on the template..
 
 Your design 

 f1

Figma Design

f2

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @d2ndjim 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [ ] linter for this project is disabled, I can see it was working fine before. Please make sure the linter is working fine and passing all the checks.
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by JuliCarracedo — APPROVED** [review]
# Status: Approved ✔️

Hi @d2ndjim, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Suggestion

- [ ] It's your portfolio after all so you don't need to do this, but I think the download icon will look better if you make it bigger.

review

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**